src/record/record.js

```diff
@@ -249,17 +250,17 @@ Record.prototype.whenReady = function( callback ) {
  */
 Record.prototype._$onMessage = function( message ) {
 	if( message.action === C.ACTIONS.READ ) {
-		this._clearTimeouts();
+		clearTimeout( this._readTimeout );
```

Comment:
Wouldn't we want to clear readAck Timeout here as well (the ACK isn't mandatory)